src/components/MainColumn.js

```diff
@@ -17,7 +17,7 @@ const MainColumn = ({ children, className }) => (
 
 MainColumn.propTypes = {
   className: PropTypes.string,
-  children: PropTypes.array,
+  children: PropTypes.node,
```

Comment:
We probably want to handle both of these cases like we do [here](https://github.com/mongodb/snooty/blob/master/src/layouts/index.js#L71)
```suggestion
  children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]).isRequired,
```